lib/oedipus_lex.rb

```diff
@@ -62,9 +55,30 @@ def to_ruby state, predicates, exclusive
         when /^:/, "nil" then
           "  [:state, #{action}]"
         else
+          uses_text = true
           "  #{action} text"
         end
 
+      if uses_text or (action and action.include? 'text')
```

Comment:
Can you help me understand the `'text'` part? I changed this to be `action.starts_with? '{'` and it passes as well. Is there a difference between the two?